include/dev/HUDL.hpp

```diff
@@ -40,18 +42,31 @@ class HUDL {
     void initLCD();
 
     /**
-         * Gets the object dictionary
-         *
-         * @return an object dictionary
-         */
-    CO_OBJ_T* getObjectDictionary();
+     * Get a pointer to the start of the object dictionary
+     *
+     * @return Pointer to the start of the object dictionary
+     */
+    CO_OBJ_T* getObjectDictionary() override {
+        return &objectDictionary[0];
```

Comment:
I get why you put this here, but I really don't like having this in the hpp. We don't do this for any other functions in EVT-core, and this isn't how it's done in the samples.

Reply:
I would argue that this is fine for a basic getter or setter such as this. One reason for using an hpp file in the first place is to include annoying code like this with the declaration. Now definitely a lot of code should NOT go in the hpp file, but for something trivial with less then 5 lines, it makes sense.

Reply:
I would argue that a method's implementation being trivial is only further reason for it not to be in the header file because that's what we want people to read when they're trying to understand the class. That being said, I do see how keeping it in one place can be convenient. I don't want to go through the effort of updating every project, but if you really want this to be the new standard for doing this, you should update the samples in EVT-core. Changes will slowly propagate from there